src/cljs/harja/views/hallinta.cljs

```diff
@@ -23,126 +23,168 @@
-     [toteumatyokalu-nakyma/simuloi-toteuma])
-
-   "Työmaapäiväkirjatyökalu"
-   :tyomaapaivakirjatyokalu
-   (when (and (istunto/ominaisuus-kaytossa? :toteumatyokalu)
-           (oikeudet/voi-kirjoittaa? oikeudet/hallinta-toteumatyokalu))
-     ^{:key "tyomaapaivakirjatyokalu"}
-     [paivakirjatyokalu-nakyma/simuloi-tyomaapaivakirja])
-
-   "Koulutusvideot"
-   :koulutusvideot
-   (when (oikeudet/voi-kirjoittaa? oikeudet/hallinta-koulutusvideot)
-     ^{:key "koulutusvideot"}
-     [koulutusvideot/nakyma])
-
-   "Palauteluokitukset"
-   :palauteluokitukset
-   (when (oikeudet/hallinta-palautevayla)
-     ^{:key "palauteluokitukset"}
-     [pl/palauteluokitukset])
-
-   "Viestitestaus"
-   :viestitestaus
-   (when true
-     ^{:key "viestitestaus"}
-     [viestinakyma/viestitestaus])
-
-   "MHU Tehtäväryhmät ja tehtävät"
-   :tehtavatjatehtavaryhmat
-   (when true
-     ^{:key "tehtavaryhmatjatehtavat"}
-     [tehtava-nakyma/tehtavat])
-
-   "MHU tarjoushinnat"
-   :mhu-tarjoushinnat
-   (when (oikeudet/hallinta-tarjoushinnat)
-     ^{:key "mhu-tarjoushinnat"}
-     [tarjoushinnat/tarjoushinnat])])
+   "Urakkatiedot"
+   :hallinta-urakat
+   ^{:key "urakoiden-hallinta"}
+   [bs/tabs {:style :tabs :classes "tabs-taso2"
+             :active (nav/valittu-valilehti-atom :hallinta-urakat)}
+
+    "Indeksit"
+    :indeksit
+    (when (oikeudet/hallinta-indeksit)
+      ^{:key "indeksit"}
+      [i/indeksit-elementti])
+
+
+    "Tehtävät"
+    :tehtavat
+    (when (oikeudet/hallinta-tehtavat)
+      ^{:key "tehtävät"}
+      [tp/toimenpidekoodit])
+
+
+    "Välitavoitteet"
+    :valtakunnalliset-valitavoitteet
+    (when (oikeudet/hallinta-valitavoitteet)
+      ^{:key "valtakunnalliset-valitavoitteet"}
+      [valitavoitteet/valitavoitteet])
+
+    "Urakkanimet"
+    :lyhytnimet
+    (when (oikeudet/hallinta-indeksit) ; ei uutta roolia roolit exceliin
+      ^{:key "lyhytnimet"}
+      [lyhytnimet/urakoiden-lyhytnimet])
+
+    "Lämpötilat"
+    :lampotilat
+    (when (oikeudet/hallinta-lampotilat)
+      ^{:key "lämpötilat"}
+      [lampotilat/lampotilat])
+
+    "Vesiväyläurakat"
+    :vesivayla-hallinta
+    (when (and (istunto/ominaisuus-kaytossa? :vesivayla)
+            (oikeudet/hallinta-vesivaylat))
+      ^{:key "vesivaylaurakat"}
+      [vu/vesivayla-hallinta])
+
+    "MHU Tehtäväryhmät ja tehtävät"
+    :tehtavatjatehtavaryhmat
+    (when true
+      ^{:key "tehtavaryhmatjatehtavat"}
+      [tehtava-nakyma/tehtavat])
+
+    "MHU tarjoushinnat"
+    :mhu-tarjoushinnat
+    (when (oikeudet/hallinta-tarjoushinnat)
+      ^{:key "mhu-tarjoushinnat"}
+      [tarjoushinnat/tarjoushinnat])
+
+    "Rahavaraukset"
+    :rahavaraukset
+    (when (oikeudet/hallinta-tarjoushinnat) ;; todo: vaihda hallinta-rahavaraukset
+      ^{:key "rahavaraukset"}
+      [rahavaraukset/rahavaraukset])
+
+    "Rahavarausten tehtävät"
+    :rahavarausten-tehtavat
+    (when (oikeudet/hallinta-tarjoushinnat) ;; todo: vaihda hallinta-rahavaraukset
```

Comment:
Täälläkin näitä todo kommentteja, ovatko ajan tasalla? Eli oikeustarkastus täytyy näemmä vielä korjata?

Reply:
Vaihdettu käyttämään oikeaa käyttöoikeutta ja poistettu todo